SingularityService/src/main/java/com/hubspot/singularity/resources/RequestResource.java

```diff
@@ -149,7 +149,11 @@ public SingularityRequestParent submit(@ApiParam("The Singularity request to cre
 
     final long now = System.currentTimeMillis();
 
-    requestManager.activate(newRequest, maybeOldRequest.isPresent() ? RequestHistoryType.UPDATED : RequestHistoryType.CREATED, now, JavaUtils.getUserEmail(user));
+    if (maybeOldRequestWithState.isPresent() && maybeOldRequestWithState.get().getRequest().isLongRunning()) {
```

Comment:
Actually, not as explicitly as I remembered, `testSchedulerHandlesFinishedTasks` uses requestResource.submit to cause tasks to be pending, so slightly different